dietpi/dietpi-software

```diff
@@ -2557,6 +2563,13 @@ _EOF_
 			fi
 		}
 
+		# $1: Software name, so config can be created and removed per-software
+		Enable_IP_forwarding()
+		{
+			echo -e 'net.ipv4.ip_forward=1\nnet.ipv6.conf.default.accept_ra=2\nnet.ipv6.conf.all.accept_ra=2\nnet.ipv6.conf.default.forwarding=1\nnet.ipv6.conf.all.forwarding=1' > "/etc/sysctl.d/dietpi-$1.conf"
+			sysctl net.ipv4.ip_forward=1 net.ipv6.conf.default.accept_ra=2 net.ipv6.conf.all.accept_ra=2 net.ipv6.conf.default.forwarding=1 net.ipv6.conf.all.forwarding=1
+		}
+
```

Comment:
Should we at least give a notification line on this step?

```
[  OK  ] DietPi-Software | APT install for: tailscale
net.ipv4.ip_forward = 1
net.ipv6.conf.default.accept_ra = 2
net.ipv6.conf.all.accept_ra = 2
net.ipv6.conf.default.forwarding = 1
net.ipv6.conf.all.forwarding = 1
```

Reply:
Done